src/cpu/jit_avx512_common_gemm_f32.cpp

```diff
@@ -63,7 +63,7 @@ struct jit_avx512_common_gemm_f32::xbyak_gemm : public jit_generator {
         auto ARG_N = abi_param2;
         auto K = abi_param3;
         auto ARG_ALPHA = abi_param4;
-#ifdef _WIN32
+#if defined(WIN32) && !defined(__GNUC__)
```

Comment:
I'm not sure I understand this part. If MinGW uses Windows x64 ABI, then we need to generate the same prologue regardless of the compiler.

Reply:
Thank you for pointed out.I think you're right.

I was careless on that.
After I accidentally undefined  `OFFSET_SHADOWSPACE`, I just chose the code just passed the compilation without thinking properly.

I'll add a commit for properly fixing the issue.